server/apps/research/admin/article_admin.py

```
@@ -1,8 +1,11 @@
 from django.contrib import admin
 from django import forms
-from apps.research.models import Article, Author
+from apps.research.models import Article, Author, ArticleSlugHistory
```

Ruff (0.8.0)

`apps.research.models.Author` imported but unused

Remove unused import: `apps.research.models.Author`

(F401)